.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl
+
+    - name: Install krew
+      shell: bash
+      run: |
+        cd "$(mktemp -d)" &&
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
```

Comment:
similarly if this can't access ${KREW_VERSION} it will not fail.

I want to see `set -euo pipefail` everywhere more than a single command is executed, or there are pipes or there are variables in the `run:` block.

Reply:
I already added it.